.github/workflows/publish.yml

```diff
@@ -12,6 +12,7 @@ env:
   GH_TOKEN: ${{ secrets.GH_TOKEN }}
   PRODUCTION: ${{ secrets.PRODUCTION }}
   SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```

Comment:
Do we want to call the value of the secret `NODE_AUTH_TOKEN `too? For consistency? I can do that.

Reply:
yes, good idea

Reply:
I added the same secret in `NODE_AUTH_TOKEN` and removed `NPM_AUTH_TOKEN`